flink-python/pyflink/fn_execution/state_impl.py

```diff
@@ -798,6 +857,11 @@ def __init__(self,
             state_handler, map_state_read_cache_size)
         from pyflink.fn_execution.coders import FlattenRowCoder
         self._key_coder_impl = FlattenRowCoder(key_coder._field_coders).get_impl()
+        self.window_coder = window_coder
+        if window_coder:
```

Comment:
It would be better if we rename it to `namespace_coder` and its implementation to `_namespace_coder_impl` to make the meaning clearer.

Reply:
Make senses.